openpype/pipeline/workfile/workfile_template_builder.py

```diff
@@ -937,7 +984,7 @@ class PlaceholderItem(object):
 
     def __init__(self, scene_identifier, data, plugin):
         self._log = None
-        self._scene_identifier = scene_identifier
+        self.name = scene_identifier
```

Comment:
Why this has changed?

Reply:
Don't put back the `name` attribute. Its very unintuitive from api point of view and it was changed on purpose.

Reply:
fixed